app/jobs/disability_claim/document_upload.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class DisabilityClaim::DocumentUpload < ActiveJob::Base
+  def perform(filename, auth_headers, user_uuid, claim_id, tracked_item_id)
+    client = EVSS::DocumentsService.new(auth_headers)
+    uploader = DisabilityClaimDocumentUploader.new(user_uuid, tracked_item_id)
+    uploader.retrieve_from_store!(filename)
+    client.upload(filename, uploader.read, claim_id, tracked_item_id)
```

Comment:
I think it would be good to have some specs around this worker. Then wherever it is triggered you can have specs that just check to make sure the worker was invoked with the appropriate arguments you are passing to it here.